### ISSUE TITLE
"skipnav" function failing to function as expected

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/freedom#41]
The skipnav button, "Skip to main content", does not function as expected when activated. It does not seem to perform the skip function (tested on Firefox, Chrome, and Safari on MacOS via mouse and keyboard). Also, attempting to activate the button appears to disrupt the focus order of the page.

This is a useful function to ensure accessibility for keyboard users (allowing for upper level and menu links to be skipped, etc.) and should be maintained, if possible. See [WAI](https://www.w3.org/WAI/WCAG22/Techniques/general/G1)'s and [WebAIM](https://webaim.org/techniques/skipnav/)'s guidelines on skipnav for more.

**Comment by aviannamiller** [comment]
As mentioned in [omeka/omeka-s#2283](https://github.com/omeka/omeka-s/issues/2283), the skip link error appears to actually be a core issue, though focus order may still need be considered in respect to the theme after link activation.

**Comment by zerocrates** [comment]
The skipnav link and the definition of the content div's id are both in the layout partial, which is theme-specific. So there probably isn't a core aspect here, but it could be a widespread mistake across multiple themes.

**Comment by aviannamiller** [comment]
That makes sense, thanks! This theme uses `id="main-content"` for its main content area, so the skipnav hyperlink should be corrected accordingly (to `href="#main-content"`).
